proto/scene-format.proto

```diff
@@ -0,0 +1,77 @@
+syntax = "proto3";
+
+package com.github.computer_graphics_course;
+
+message Scene {
+	enum MaterialType {
+		SOLID = 1;
+		GLASS = 2;
+		// other materials
+	}
+	message Vector3 {
+		double x = 1;
+		double y = 2;
+		double z = 3;
+	}
+	message Color {
+		double r = 1;
+		double g = 2;
+		double b = 3;
+	}
+	message Transform {
+		int32 id = 1;
```

Comment:
I think we decided to use object id for `parent_id`. If so, the `id` field in `Transform` seems to have no use.